ale/base/data_naif.py

```diff
@@ -322,23 +322,77 @@ def sensor_position(self):
                 # spkezr returns a vector from the observer's location to the aberration-corrected
                 # location of the target. For more information, see:
                 # https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/FORTRAN/spicelib/spkezr.html
-                state, _ = spice.spkezr(self.target_name,
+                state, _ = spice.spkezr(self.spacecraft_name,
                                        time,
                                        self.reference_frame,
                                        self.light_time_correction,
-                                       self.spacecraft_name,)
+                                       self.target_name,)
                 pos.append(state[:3])
                 vel.append(state[3:])
             # By default, spice works in km, and the vector returned by spkezr points the opposite
             # direction to what ALE needs, so it must be multiplied by (-1)
-            self._position = [p * -1000 for p in pos]
-            self._velocity = [v * -1000 for v in vel]
+            self._position = [p * 1000 for p in pos]
+            self._velocity = [v * 1000 for v in vel]
         return self._position, self._velocity, self.ephemeris_time
 
+    @property
+    def frame_trace(self):
+        frame_codes = [self.sensor_frame_id]
```

Comment:
Can you make this a separate function where the starting frame ID is passed in. We will need to do this for both the sensor frame and the body frame

Reply:
:+1: I agree that this probably shouldn't live in the data_naif module